src/components/LoadingSpinner.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
don't Need to import React in every file you should remove this line and you are good to go 
```suggestion

```